### PR TITLE
scale back try_write function on unix

### PR DESCRIPTION
**Pull request by kalefranz**
@msarahan @mingwandroid 

Any issues with this PR?  Relates to https://github.com/conda/conda/issues/2151

### CODE REVIEW

**Comment by kalefranz** [comment]
+ @wulmer

File: conda/utils.py [hunk 1]
```diff
@@ -1,5 +1,7 @@
 from __future__ import print_function, division, absolute_import
 
+from genericpath import isfile
```
Comment (wulmer) [inline conda/utils.py]: Shouldn't this import be added to line 10 ?

```python
from os.path import abspath, isdir, isfile, join
```

File: conda/utils.py [hunk 2]
```diff
@@ -89,18 +91,24 @@ def gnu_get_libc_version():
     return f()
 
 
-def try_write(dir_path):
+def try_write(dir_path, heavy=False):
     if not isdir(dir_path):
```
Comment (wulmer) [inline conda/utils.py]: IMHO a docstring might be necessary now, explaining the arguments.

**Comment by wulmer** [comment]
Looks good! Good idea to take the shortcut on Linux.

File: conda/utils.py (hunk 1 above)
Comment (kalefranz) [inline conda/utils.py]: oops. Blame my IDE.

File: conda/utils.py (hunk 2 above)
Comment (kalefranz) [inline conda/utils.py]: good call

**Pull request merged**